src/NUnitConsole/nunit3-console/ResultSummary.cs

```diff
@@ -181,21 +182,30 @@ private void Summarize(XmlNode node)
                     switch (status)
                     {
                         case "Passed":
-                            PassCount++;
+                            if (failedInFixtureTearDown)
+                                FailureCount++;
+                            else
+                                PassCount++;
                             break;
                         case "Failed":
-                            if (label == null)
+                            if (label == null || failedInFixtureTearDown)
```

Comment:
To maintain compatibility with the way (method) TearDown works, I think that failedInFixtureTearDown should increase the Error count. See my general comment.

Reply:
You are right. Once more I got error vs failure wrong (perhaps since the xml elements are called `failure` or since the test "failed with an error"). I'll change this.

Reply:
Yes... both Failure and Error are a subset of status Failed... confusing for sure. Maybe we should have had status Good, Bad, Indifferent. :-)

Reply:
Also, I think you could get this wrong and nobody would be likely to notice. For example, most mock framework failures are reported as errors.

Reply:
I've changed the code such that the Error count is increased.